### PR TITLE
Add scripts for swagger-ui

### PR DESCRIPTION
**Pull request by soyuka**
see https://github.com/api-platform/core/pull/768

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -78,5 +78,16 @@
         "branch-alias": {
             "dev-master": "2.0-dev"
         }
+    },
+    "scripts": {
```
Comment (teohhanhui) [inline composer.json]: We cannot have duplicate `scripts` key. There's already an existing section above with the Symfony scripts.

**Comment by soyuka** [comment]
Done

**Comment by soyuka** [comment]
This has to be merged in combination with api-platform/core#768

**Comment by Simperfit** [comment]
Could you check why the test are not green :p ?

**Comment by soyuka** [comment]
May it has to be with the fact that  api-platform/core#768 isn't available yet?

**Pull request closed without merging**